kubernetes/llama-stack/deployment.yaml

```diff
@@ -37,13 +37,13 @@ spec:
             - name: VLLM_MAX_TOKENS
               value: '128000'
             - name: LLAMA3B_MODEL
-              value: meta-llama/Llama-3.2-3B-Instruct
+              value: llama32-3b
             - name: GRANITE_URL
-              value: 'https://granite-8b-llama-serve.apps.ocp-beta-test.nerc.mghpcc.org/v1'
+              value: 'https://granite32-8b.llama-serve.svc.cluster.local/v1'
             - name: GRANITE_MODEL
-              value: ibm-granite/granite-3.2-8b-instruct
+              value: granite32-8b
             - name: LLAMA3B_URL
-              value: 'https://llama32-3b-llama-serve.apps.ocp-beta-test.nerc.mghpcc.org/v1'
+              value: 'https://llama32-3b.llama-serve.svc.cluster.local/v1'
```

Comment:
Can we move the `LLAMA3B_URL` above the `LLAMA3B_MODEL` to keep things consistent?